README.md

```diff
@@ -56,21 +55,42 @@ Import the `authenticator-browser-extension` in the [`protractor.conf.js`](https
 ```javascript
 // protractor.conf.js
 
-const { Authenticator } = require('authenticator-browser-extension');
+const { Authenticator } = require("authenticator-browser-extension");
 
 exports.config = {
+  capabilities: {
+    browserName: "chrome",
 
-    capabilities: {
-        browserName: 'chrome',
-
-        chromeOptions: {
-            extensions: [
-                Authenticator.for('username', 'password').asBase64()
-            ]
-        }
+    chromeOptions: {
+      extensions: [Authenticator.for("username", "password").asBase64()],
     },
+  },
+
+  // other Protractor config
+};
+```
+
+### Puppeteer
+
+Import the `authenticator-browser-extension` and load `Authenticator` extension while launching the puppeteer browser:
+
+```javascript
+const { Authenticator } = require('authenticator-browser-extension');
+
+await puppeteer.launch({
+      headless: false,
+
+      args: [
+        `--disable-extensions-except=${Authenticator.for(
+          "username",
+          "password"
+        ).asFileAt("pathToFolder")}`, //Authenticator will generate necessary files in pathToFolder
```

Comment:
I'd use `.asFileAt('./path/to/folder')` in the example to indicate that it's a relative path

Reply:
changed as suggested